modules/KalturaSupport/resources/mw.kAnalony.js

```diff
@@ -550,17 +550,16 @@
             var pageReferrer =  statsEvent[ 'referrer' ];
             var queryPos = pageReferrer.indexOf("?");
             if (queryPos > 0) {
-                pageReferrer = pageReferrer.substring(0, queryPos);
+              pageReferrer = pageReferrer.substring(0, queryPos);
             }
 
-            var encodedReferrer = encodeURIComponent(pageReferrer);
```

Comment:
@shalommeoded the first test was OK, you need to test it with `encodeURIComponent`

Reply:
@OrenMe I have re-added it.